docs/UserGuide.md

```diff
@@ -275,7 +275,7 @@ You can import patient data from a csv file.
 Format: `import i/./data/FILENAME.csv`
 
 - The csv file should be formatted in a way that it only contains 4 columns with specific headings as shown above.
```

Comment:
Can you add " Please ensure all entries are present. If the patient row has missing information, enter "NIL" below that column".